tests/files/api_fixtures/endpoint_operations.yaml

```diff
@@ -34,6 +34,14 @@ transfer:
         "display_name": "myserver",
         "organization": "My Org",
         "username": "auser",
+        "canonical_name": "auser#myserver",
+        "contact_email": null,
+        "contact_info": null,
+        "department": null,
+        "info_link": null,
+        "keywords": null,
+        "local_user_info_available": true,
+        "owner_string": "auser@globusid.org",
```

Comment:
These are the missing fields for `show`. I wasn't sure where in the documents to put them, so I put them in the same place each time, in alphabetical order.

Transfer actually sorts its output so that the order is fixed, so I might come back through here and sort these later, but I didn't want to mix that into this change.

Reply:
This seems fine for now.
It would be nice if everything were always alphabetized Transfer side, not sure how big of a change that would be

Reply:
I think Transfer does produce sorted output, but then we load-and-dump that through the python `json` module without being careful to preserve order. (I _think_ newer versions of cpython preserve order as an implementation detail, but I'm not even sure offhand.)